composer.json

```diff
@@ -42,13 +42,13 @@
     "laravel/tinker": "^2.6",
     "laravel/ui": "^3.3",
     "laravelcollective/html": "^6.2",
-    "lcobucci/clock": "1.2.0",
-    "lcobucci/jwt": "^3.4.5",
+    "lcobucci/clock": "1.2.0|2.0.0",
+    "lcobucci/jwt": "^3.4.5|^4.0.4",
```

Comment:
It gets upgraded in the lockfile to 4.1.5

There are no documented backwards-compatibility breaks with this version.

Reply:
Also not used directly by us; mostly used indirectly by Laravel.